scripts/testpop/testpop.go

```diff
@@ -57,14 +60,33 @@ func main() {
 	if *verbose {
 		logger.Level = log.DebugLevel
 	}
-	rp, _ := ringpop.New("ringpop",
-		ringpop.Channel(ch),
+
+	options := []ringpop.Option{ringpop.Channel(ch),
 		ringpop.Identity(*hostport),
 		ringpop.Logger(bark.NewLoggerFromLogrus(logger)),
-		ringpop.SuspectPeriod(5*time.Second),
-		ringpop.FaultyPeriod(5*time.Second),
-		ringpop.TombstonePeriod(5*time.Second),
-	)
+		ringpop.SuspectPeriod(5 * time.Second),
+		ringpop.FaultyPeriod(5 * time.Second),
+		ringpop.TombstonePeriod(5 * time.Second)}
+
+	if *stats != "" {
+		var statsdClient statsd.Statter
+		var err error
+		if statsPattern.MatchString(*stats) {
+			statsdClient, err = statsd.New(*stats, "")
```

Comment:
Should it then be `--stats-file` and `--stats-udp` to disambiguate between the two?